### ISSUE TITLE
Vendor related feature request - downloading cookbook files

### ISSUE THREAD

**Issue by HenrikSkriver** [berkshelf/berkshelf#604]
Hi
This is not an issue/bug but a feature request. I do not know where else to put here or get input on how to solve the issue. Please redirect me if this is misplaced.

My problem is that I am to ship a xen-server image which has the chef-server, knife, etc. installed and also the cookbooks which are to be installed on some clients.
However, at the customer site is 100% offline.
So for instance, lets say I am creating a simple cookbook which is ONLY to target an ubuntu 12.04 platform and that cookbook simply installs java 7 using the opscode java cookbook.
In this case I would like to be able to download the actual jdk jar file before shipping the chef-server image to the customer.
For instance as part of the berks install --path vendor/cookbooks step.
Basically I want to bundle everything nice and tidy.
Any ideas?
Either I am missing some aspect of how to use berkshelf to bundle my cookbooks or I am thinking that berkshelf only gets me halfway there wrt. packing up dependencies.

Thank you.
Regards Henrik
